app/models/config.rb

```diff
@@ -19,7 +19,8 @@ class Config
     resources_url: 'A link to a Google Drive folder with CM resources. ' \
                    'Ex: https://drive.google.com/drive/my-resource-dir',
     fax_service: 'A link to your fax service. ex: https://www.efax.com',
-    start_of_week: "The start day of your budget week, if it doesn't start on Monday. ex: Sunday."
+    start_of_week: "The start day of your budget week, if it doesn't start on Monday. ex: Sunday.",
+    show_practical_support: 'Toggle the visibility of practical support information. Enter "Yes" to show or "No" to hide. Note: This will not remove any existing data.'
```

Comment:
This descriptive text isn't quite what's going on in the code. I would suggest:

```suggestion
    show_practical_support: 'Enter "no" to hide the Practical Support panel on patient pages. This will not remove any existing data.'
```

(or the equivalent of this being a negative per comment above)